packages/plugin-e2e/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "@grafana/plugin-e2e",
+  "version": "0.0.1",
+  "private": true,
+  "main": "./dist/cjs/index.js",
+  "types": "./dist/cjs/index.d.ts",
```

I think these properties can now be

```suggestion
  "main": "./dist/index.js",
  "types": "./dist/index.d.ts",
```